packages/holocron/src/holocronModule.jsx

```diff
@@ -17,27 +17,88 @@ import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
 import { createSelector } from 'reselect';
 import hoistStatics from 'hoist-non-react-statics';
-import { INIT_MODULE_STATE } from './ducks/load';
 
 import {
   LOAD_KEY,
   REDUCER_KEY,
   MODULES_STORE_KEY,
-} from './constants';
+  INIT_MODULE_STATE,
+} from './ducks/constants';
+
+// Execute deprecated load function and provide deprecation message
+export function executeLoad({ load, ...restProps } = { }) {
+  if (load) {
+    console.warn('The \'load\' function in holocron has been deprecated. Please use \'loadModuleData\' instead.');
+    return load(restProps);
+  }
+  return undefined;
+}
```

Comment:
Idempotent function to call `load` and `warn` with deprecation message.

Reply:
like what?

Reply:
deprecating the load function, just wondering if there was a use-case we missed

Reply:
None that I can see. `loadModuleData` provides a superset of functionality in comparison to `load`. See the API docs to compare the args. The integration tests are satisfied in the One App PR.